src/App/src/app/app.component.html

```diff
@@ -0,0 +1,8 @@
+<div>
+    <h3 class="h3-head">Label binding</h3>
+    <label-planetext></label-planetext>
```

Comment:
Do we really need label-planetext component?

Reply:
In case we will have to provide any additional logic to label we need to separate it